cds/modules/deposit/static/json/cds_deposit/forms/project.json

```diff
@@ -93,10 +93,17 @@
         "inline": true,
         "items": [
           {
+            "type": "uiselect",
             "key": "contributors[].name",
-            "placeholder": "Name",
-            "description": "",
-            "notitle": true
+            "notitle": true,
+            "placeholder": "Author name",
+            "options": {
+              "asyncCallback": "$ctrl.autocompleteAuthors",
+              "url": "https://cds.cern.ch/submit/get_authors",
```

Comment:
we'll always be on ``https`` on production, but for developments could you try if ``//cds.cern.ch/submit/get_authors`` works?